invenio_app_ils/relations/api.py

```diff
@@ -294,3 +293,64 @@ def remove(self, pid):
             self._recreate_relations_with_random_parent(
                 all_relations, pids_to_relate
             )
+
+
+class LinkedRelation(Relation):
+    """Relation class for linked type."""
+
+    def children_of(self, pid):
+        """Get all children PID for relations where the given PID is parent."""
+        return [r.child for r in self.get_relations_by_parent(pid)]
+
+    def parents_of(self, pid):
+        """Get all parents PID for relations where the given PID is child."""
+        return [r.parent for r in self.get_relations_by_child(pid)]
+
+    def relation_exists(self, parent_pid, child_pid):
```

Comment:
Here you override the parent `relation_exists` method: is this on purpose? Because I think it is more correct to check for both A -> B and B -> A.